test/service/actions/posts.test.js

```diff
@@ -156,19 +156,23 @@ describe('Actions.Posts', () => {
                 channelId
             )(store.dispatch, store.getState);
 
+            let shouldStop = false;
```

Comment:
What's this for? Shouldn't this just terminate once done is called?

If we're having multiple changes to the store due to the websocket, maybe we should redo these tests so that they don't use subscribe and instead work like
```
// SETUP

await Actions.doSomething()(store.dispatch, store.getState);

// ASSERTS
```

since right now these tests feel incredibly fragile. This would also let us change the test themselves from using a callback to being async functions so we wouldn't need to call done manually.